src/site/content/en/authors/feed.njk

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Note that we may want to do this (https://github.com/GoogleChrome/web.dev/issues/2652) in this PR. Or do a follow up PR to do it for all of the feeds and other files.